app/sidekiq/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -103,7 +103,6 @@ def perform(submission_id) # rubocop:disable Metrics/MethodLength
           user_account = UserAccount.find_by(id: submission.user_account_id) ||
                          Account.find_by(idme_uuid: submission.user_uuid)
 
-          user = OpenStruct.new({ user_account_uuid: user_account.id, flipper_id: user_account.id })
           begin
             submit_to_claims_api = submission.claims_api?
```

Comment:
this can be combined with line 109, it is only used once.

Reply:
`submit_to_claims_api = submission.claims_api?`

Reply:
@tblackwe no, we determined that that assignment was important (@freeheeling identified this on [thread](https://dsva.slack.com/archives/C04U3EDUR3M/p1718224306419789?thread_ts=1718063256.233939&cid=C04U3EDUR3M). In fact I initially tried that and it was breaking tests.

Reply:
@sethdarragile6, that was before `options = generate_options_hash(submit_to_claims_api, user)` was removed in this PR. The assigned variable isn't needed as that was the only other instance in which it was used.

Reply:
ohhhhh I get now. Sorry, guys. Fixing now...